uelc/main/tests/test_models.py

```diff
@@ -1,6 +1,63 @@
 from django.test import TestCase
+#from django.core.exceptions import ValidationError
```

Comment:
I like having that in there because I am pretty sure I am going to need it soon. But I will delete it for now.